linux_service/start_moneo_services.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+PUBLISHER=$1
+
+systemctl start moneo@node_exporter.service
+systemctl start moneo@net_exporter.service
+systemctl start moneo@nvidia_exporter.service
+
+if [[ ! -z "$PUBLISHER" ]];
+then
+    if [[ "$PUBLISHER" = true ]];
+    then
+        sleep 5 # wait a bit for the exporters to start
+        systemctl start moneo_publisher.service 
```

Comment:
what happens if publisher is started before the exporters are active?

Reply:
It tries to reconnect a few tries but this part could be a bit more robust. Right now Moneo cli and this script makes sure it has enough time.